csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/ISquidexClientManager.cs

```diff
@@ -0,0 +1,35 @@
+// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System.Collections.Generic;
+using System.Net.Http;
+using Squidex.ClientLibrary.Management;
+
+namespace Squidex.ClientLibrary
+{
+    public interface ISquidexClientManager
+    {
+        SquidexOptions Options { get; }
```

Comment:
Can you just add one line between each member?

Reply:
Hi,
I created a new commit and added new lines between all members in the interface